coalib/settings/ConfigurationGathering.py

```diff
@@ -135,27 +135,30 @@ def warn_nonexistent_targets(targets, sections, log_printer):
 
```

Comment:
It doesnt give context.  and `exchangeable` is too big a word for this.  simplify.